admin_guide/managing_networking.adoc

```diff
@@ -85,6 +85,42 @@ can now access any pods and services in the cluster and vice versa.
 Alternatively, instead of specifying specific project names, you can use the
 `--selector=<project_selector>` option.
 
+[[admin-guide-disabling-hostname-collision]]
+== Disabling Host Name Collision Prevention For Ingress Objects
+
+In {product-title}, host name collision prevention for routes and ingress
+objects is enabled by default. This means that the host name in a route or
+ingress object can only be set on creation and not edited afterwards. Disabling
+host name collision prevention lets you edit a host name after creation.
+However, because {product-title} uses the route creation timestamp to determine
+the oldest route or ingress object for a given host name, the route or ingress
+object can hijack a host name with a newer route. This can happen if an older
+route changes its host name, or if an ingress object is introduced.
+
+This is relevant to {product-title} installations that depend upon Kubernetes
+behavior, including allowing the host names in ingress objects be edited. You
+can disable host name collision to enable system-wide routing through ingress
+objects.
```

Comment:
this does not enable routing through ingress objects just editing of the hostname field